rust/scx_utils/src/topology.rs

```diff
@@ -449,10 +463,24 @@ fn create_insert_cpu(cpu_id: usize, node: &mut Node, online_mask: &Cpumask) -> R
         span: Cpumask::new()?,
     });
 
+    let core_type = match avg_cpu_freq {
+        Some(avg_freq) => {
+            if max_freq > avg_freq {
+                CoreType::Big
+            } else if max_freq > 0 && max_freq < avg_freq {
+                CoreType::Little
+            } else {
+                CoreType::Big
+            }
+        }
+        None => CoreType::Unknown,
```

Comment:
Do we need unknown? Wouldn't the scheduler treat it as `Big` anyway?

Reply:
e.g. maybe just printing out a warning when parsing fails and hiding the complexity from the schedulers would be better?

Reply:
Yeah, should always fallback to big. Doesn't make sense to add extra complexity to handle that case.